test/e2e/tests/test_hosted_zone.py

```diff
@@ -14,18 +14,13 @@
 """Integration tests for the Route53 HostedZone resource
```

Comment:
Changes in this file are essentially the same, just refactoring some parts because there was a lot of common code with `test_record_set.py`